examples/Cliner/pipeline.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The main running pipeline for the rewriter.
+"""
+import sys
+
+from examples.Cliner.Cliner import ClinicalNER
+from examples.Cliner.Cliner_train import CliTrain
+from forte.data.data_pack import DataPack
+from forte.data.readers import RawDataDeserializeReader
+from forte.pipeline import Pipeline
+
+# Let's create a pipeline first.
+pipeline = Pipeline[DataPack]()
+
+
+def do_process(input_pack_str: str):
+    datapack: DataPack = pipeline.process([input_pack_str])
+    data_json = datapack.serialize()
+    with open('generation.json', 'w') as fo:
```

Comment:
So here you basically implement the writer yourself.  I think this is probably the reason for double json encoding as well. 

Actually, for serialization you can directly call a writer without doing this. You can follow the example here:

https://github.com/asyml/forte/blob/master/examples/serialization/serialize_example.py#L104

Reply:
I have changed it to call the writer, while it is written to the output/cliner_input.json instead of a new generation.json. I think it doesn't matter.

I will check the code again according to the json and then open a new branch.

Reply:
So the writer, when provided correct parameters, will write out documents in that directory using its "pack_name". So if you set the `pack_name` somewhere it will result in that file name.  https://github.com/asyml/forte/blob/dbf04edb3e7e58e59022e324b87a182c26c93de7/forte/processors/writers.py#L31